packages/runner-docker/Dockerfile

```diff
@@ -11,11 +11,9 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Setup @mmarchini/observe
-RUN <<-EOF
-  npm install -g @mmarchini/observe@^3.0.0
-  chown -R node:node /usr/local/lib/node_modules
-  chown -R node:node /usr/local/bin/observe
-EOF
+RUN npm install -g @mmarchini/observe@^3.0.0 && \
+    chown -R node:node /usr/local/lib/node_modules && \
+    chown -R node:node /usr/local/bin/observe
```

Comment:
IIRC I did this because there is an error mode that makes this form harder to debug. Is this required for windows compat for some reason?

Reply:
It is not that it *causes* an error. IIRC I broke them apart because when there is an error the message is harder to read and debug.